.github/dependabot.yml

```diff
@@ -0,0 +1,22 @@
+version: 2
+updates:
+- package-ecosystem: github-actions
+  directory: "/"
+  schedule:
+    interval: weekly
+  open-pull-requests-limit: 10
+- package-ecosystem: gomod
+  directory: "/server"
+  schedule:
+    interval: weekly
+  open-pull-requests-limit: 10
```

Comment:
We also need to monitor the `/operator` folder

Reply:
Fixed in `abbcc0c`.